src/test/java/org/folio/edge/patron/utils/PatronMockOkapi.java

```diff
@@ -80,7 +82,9 @@
   public static final String goodRequestId = holdCancellationHoldId ;
   public static final String nonUUIDHoldCanceledByPatronId = "patron@folio.org";
   public static final String patronComments = "Can you deliver this to the History building for Professor Grant?";
-  public static final String wrongOffsetMessage = "'offset' parameter is incorrect. parameter value {%s} is not valid: must be an integer, greater than or equal to 0";
+  public static final String wrongIntegerParamMessage = "'%s' parameter is incorrect. parameter value {%s} is not valid: must be an integer, greater than or equal to 0";
```

Comment:
Instead of duplicating this constant in the test (along with `OFFSET` & `LIMIT`, can we just make the constants in `PatronHandler` public so we can reference them via static imports?  This way, if we need to change these constants we only have to do so in one place.

Reply:
Changed